MedicalDocManagment.WebUI/Controllers/AdminController.cs

```diff
@@ -281,6 +281,17 @@ public IHttpActionResult GetPosition(int id)
         }
         #endregion
 
+        #region Mkh's methods
+
+        [HttpGet]
+        public IHttpActionResult GetClassesMkh()
```

Comment:
Consider moving this method into separate controller, since it should be used only from main application.